From 5a97eca43ae3f7c0472d33f4cedf9e955ceed2ba Mon Sep 17 00:00:00 2001
From: Igor Opaniuk <igor.opaniuk@foundries.io>
Date: Tue, 15 Feb 2022 18:19:08 +0200
Subject: [PATCH 311/322] [FIO internal] imx8: select HAS_CAAM only if OPTEE is
 n

Select HAS_CAAM symbol only if OPTEE is not selected.

Signed-off-by: Igor Opaniuk <igor.opaniuk@foundries.io>
---
 arch/arm/mach-imx/imx8/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-imx/imx8/Kconfig b/arch/arm/mach-imx/imx8/Kconfig
index 99ea88be39a..695231af779 100644
--- a/arch/arm/mach-imx/imx8/Kconfig
+++ b/arch/arm/mach-imx/imx8/Kconfig
@@ -20,7 +20,7 @@ config IMX_LOAD_HDMI_FIMRWARE_TX
 
 config IMX8
 	bool
-	select HAS_CAAM
+	select HAS_CAAM if !OPTEE
 
 config MU_BASE_SPL
 	hex "MU base address used in SPL"
-- 
2.34.1

